miso-web/src/main/webapp/WEB-INF/miso-config.xml

```diff
@@ -23,15 +23,18 @@
 -->
 
 <beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.1.xsd"
+  xmlns:context="http://www.springframework.org/schema/context"
+  xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.1.xsd
+                        http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx-3.1.xsd
+                        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd"
   default-autowire="byName">
 
+  <context:component-scan base-package="uk.ac.bbsrc.tgac.miso.service" /> 

```

Comment:
there's a component scan on `uk.ac.bbsrc.tgac.miso` set in `applicationContext.xml`, which *should* make this one redundant